CI sometimes fails

I don't know why at the moment, but the Test workflow sometimes fails with the following error:

```
/home/runner/work/duck/duck/node_modules/workerpool/src/WorkerHandler.js:207
      me.worker.ready = true;
                      ^

TypeError: Cannot set property 'ready' of null
    at Worker.<anonymous> (/home/runner/work/duck/duck/node_modules/workerpool/src/WorkerHandler.js:207:23)
    at Worker.emit (events.js:314:20)
    at Worker.EventEmitter.emit (domain.js:483:12)
    at MessagePort.<anonymous> (internal/worker.js:201:53)
    at MessagePort.emit (events.js:314:20)
    at MessagePort.EventEmitter.emit (domain.js:483:12)
    at MessagePort.onmessage (internal/worker/io.js:80:8)
    at MessagePort.exports.emitMessage (internal/per_context/messageport.js:11:10)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! chunks@1.0.0 build:ci: `duck build --batch local -n`
npm ERR! Exit status 1
```

--- a/package.json
+++ b/package.json
@@ -57,7 +57,7 @@
     "split2": "^3.2.2",
     "strip-json-comments": "^3.1.1",
     "tempy": "^1.0.0",
-    "workerpool": "^6.0.0",
+    "workerpool": "^6.1.4",
     "xmlbuilder": "^15.1.1",
     "yargs": "^16.2.0",
     "zet": "^1.1.0"

